lib/checks/autofix/autofix.go

```diff
@@ -45,7 +45,7 @@ func Fix(ctx context.Context, probes []*agentpb.Probe, progress utils.Progress)
 		if probe.Status != agentpb.Probe_Failed {
 			continue
 		}
-		if err := fixProbe(ctx, probe, progress); err != nil {
+		if err := fixProbe(ctx, probe, progress); err != nil && !trace.IsNotImplemented(err) {
```

Comment:
So if fixProbe returns "not implemented" error, the probe will be added to fixed probes?

Reply:
Hmm - good catch since I wanted to only avoid log spam in the impossible case.